includes/modules/pages/account_history_info/header_php.php

```diff
@@ -21,6 +21,9 @@
 
 require DIR_WS_CLASSES . 'order.php';
 $order = new order($_GET['order_id']);
+if ($order === false) {
```

Comment:
A class' `__construct` method, per the PHP [documentation](https://www.php.net/manual/en/language.oop5.decon.php), is implicitly defined as a `void` returning function.  There is a closed PHP RFC (https://wiki.php.net/rfc/make_ctor_ret_void) that would have made this more concrete.

Better, for potential PHP changes in the future, to use
```php
if (empty($order->info)) {
```